.travis.yml

```diff
@@ -76,9 +76,12 @@ script:
 - .build/venv/bin/alembic -c c2cgeoportal/tests/functional/alembic.ini downgrade base
 - .build/venv/bin/python setup.py nosetests --stop --nocapture --nologcapture
 - travis/status.sh
-# verify c2cgeoportal version
-- if [[ ${TRAVIS_TAG} =~ ^[0-9]+.[0-9]+.[0-9]+(\.dev[0-9]+|\.rc[0-9]+|\.[0-9]+)?$ ]] ; then test `grep "REQUIREMENTS ?= 'c2cgeoportal==" c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl|awk -F '==' '{{print $2}}'` = "${TRAVIS_TAG}'" ; fi
-- if [[ ${TRAVIS_TAG} =~ ^[0-9]+.[0-9]+.[0-9]+(\.dev[0-9]+|\.rc[0-9]+|\.[0-9]+)?$ ]] ; then test `python setup.py --version` = ${TRAVIS_TAG} ; fi
+- |
+    if [[ ${TRAVIS_TAG} =~ ^[0-9]+\.[0-9]+\..+$ ]] ; then
+        sed -e 's/REQUIREMENTS \?= c2cgeoportal/'"REQUIREMENTS ?= '${TRAVIS_TAG}'"'/g' -i c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl;
+        sed -e 's/version="[0-9]+\.[0-9]+",/version="'"${TRAVIS_TAG}"'"/g' -i setup.py;
```

Comment:
Are you sure there are only 2 numbers? Maybe you want to eat the rest of the version numbers until the coma...
And put back the coma